test/e2e/live_iso_test.go

```diff
@@ -68,8 +80,9 @@ var _ = Describe("Live-ISO", Label("required", "live-iso"), func() {
 					URL:        imageURL,
 					DiskFormat: pointer.String("live-iso"),
 				},
-				BootMode:       metal3api.Legacy,
-				BootMACAddress: bmc.BootMacAddress,
+				BootMode:              metal3api.Legacy,
```

Comment:
Sigh.

Reply:
One thing at a time. I have added https://github.com/metal3-io/baremetal-operator/issues/1607 for it :wink: